File: frontend/src/api/meeting_room.js
import http from './http'

export const getMeetingRooms = (params) => http.get('/meeting-rooms', { params })
export const createMeetingRoom = (data) => http.post('/meeting-rooms', data)
export const updateMeetingRoom = (id, data) => http.put(`/meeting-rooms/${id}`, data)
export const deleteMeetingRoom = (id) => http.delete(`/meeting-rooms/${id}`)

File: backend/routes/meeting_room.go
package routes

import (
	"oa-system/handlers"

	"github.com/gin-gonic/gin"
)

// meetingRoomRoutes 执行相关业务逻辑
func meetingRoomRoutes(rg *gin.RouterGroup) {
	rg.GET("/meeting-rooms", handlers.GetMeetingRooms)
	rg.POST("/meeting-rooms", handlers.CreateMeetingRoom)
	rg.PUT("/meeting-rooms/:id", handlers.UpdateMeetingRoom)
	rg.DELETE("/meeting-rooms/:id", handlers.DeleteMeetingRoom)
}
